elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -190,7 +190,7 @@ $variables: map-merge($variables, $nav-light-dom-variables);
         }
 
         [slot="trigger"] {
-            font-size: 16px;
+            font-size:  #{pfe-local($region: trigger, $cssvar: FontSize, $fallback: $trigger--FontSize--fallback)};
```

Comment:
This is a fallback style specific to the Edge browser.  See line 177 where the browser-query is being called.  Are you seeing this style show up in browsers other than Edge?

Reply:
@castastrophe sorry i forgot to check the browser query. i'll update it to 
`font-size:  #{pfe-local($region: trigger, $cssvar: FontSize)}; `

Reply:
What I'm suggesting is that we leave this as-is.  The Edge browser query is here to fix issues with cascading variables in older versions of Edge so hardcoding 16px is preferred.  @chrisdo1 can confirm this though.

Reply:
@castastrophe okay got it.

Reply:
I agree with Cassondra that we can leave it as is. The good news is that this browser mixin is for Edge 17 and 18. When we don't need to support those anymore we can remove all this code.